src/common/components/index.tsx

```diff
@@ -1,2 +1,8 @@
-export * from './Select';
```

Comment:
I would prefer to do `import { Button, Select } from '../../common/components'` rather than

```
import { Button } from '../../common/components/Button'
import { Select } from '../../common/components/Select'
```

Is there a better way to do this?